src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```

Comment:
font-style by default is normal - redudant.
Chech font-weight by design, it's mu fix your tests. For other places too